From 52b603abf05945d0e8c18dfc8917611fe9c42c8c Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Mon, 24 Dec 2018 17:54:23 +0800
Subject: [PATCH 446/706] arm64: dts: ls1028a: add properties for ENETC 1588
 node

Added clock and endianness properties for ENETC 1588
device tree node.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
(cherry picked from commit 8cbda05e7df5535b68dcda0b2dbce7c8994325c6)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index d0e77f504357..e568e283a750 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -662,6 +662,8 @@
 			};
 			pci@0,4 {
 				reg = <0x000400 0 0 0 0>;
+				clocks = <&clockgen 4 0>;
+				little-endian;
 			};
 			pci@0,5 {
 				reg = <0x000500 0 0 0 0>;
-- 
2.17.1

